src/vm/methodtablebuilder.cpp

```diff
@@ -7667,6 +7704,23 @@ MethodTableBuilder::PlaceInterfaceMethods()
                 PlaceMethodFromParentEquivalentInterfaceIntoInterfaceSlot(itfSlotIt, pCurItfEntry, &rgInterfaceDispatchMapTypeIDs, dwCurInterface);
             }
 #endif
+
+            /*
```

Comment:
I've removed the code for now. It might be useful going forward when we go for a new implementation but I can easily add it back.